lib/argonaut_web/controllers/slack_controller.ex

```diff
@@ -0,0 +1,199 @@
+defmodule ArgonautWeb.SlackController do
```

Comment:
This is an intense controller. I'd extract a lot of functions in here into a domain of some sort.